src/entities/player.lua

```diff
@@ -1,9 +1,11 @@
 local assets = require("assets")
 local player = {}
+local debug = true
 
 -- create car
 function player:load(width, height)
-  self.acceleration = 300
+  -- acceleration in m/s^2
+  self.acceleration = 6.5
```

Comment:
This is one step closer to where we oughta go with this.  I think when we read that article and look at this again we should think about a summation of forces.

Reply:
Yeah I initially started this work before I read that article, but I agree that this feels a lot more sane.